notebooks/kdd_tutorial.ipynb

```diff
@@ -431,6 +431,8 @@
         "  && wget -q https://raw.githubusercontent.com/facebookresearch/mmf/master/mmf/configs/datasets/okvqa/defaults.yaml \\\n",
         "  -O \"/content/configs/okvqa_colab.yaml\" \\\n",
         "  && sed -i \"s/okvqa:/okvqa_colab:/g\" /content/configs/okvqa_colab.yaml \\\n",
+        "  && sed -i \"s/okvqa\\//okvqa_colab\\//g\" /content/configs/okvqa_colab.yaml \\\n",
```

Comment:
Shouldn't above sed take care of all of the replacement? Should be possible to do using on sed replace.

Reply:
No it doesn't take care of all.

Reply:
confirmed that this works.